client/src/components/MobileMenu.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react'
+import { NavLink } from 'react-router-dom'
+import { withRouter } from 'react-router'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+import { Menu, ExitToApp } from '@material-ui/icons'
```

Comment:
[information about how to minimize bundle size](https://material-ui.com/guides/minimizing-bundle-size/)